components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__gender.xml

```diff
@@ -111,6 +111,9 @@
       <content>{{include reference="PhenoTips.PatientSheetMacros" /}}
 
 {{velocity}}
+#if ($xcontext.action == 'edit' &amp;&amp; $obj.getProperty('gender').value == '')
```

Comment:
The second part of the condition only works after the patient record has been saved at least once. To make it more generic, use this:

    #if ($xcontext.action == 'edit' &amp;&amp; "$!obj.getProperty('gender').value" == '')